Metro-Records/la-metro-councilmatic
# Add final adjustments for bootstrap upgrade and a11y improvements

## Overview

This continues the template upgrade to bootstrap 5, this time for miscellaneous small changes throughout various templates. Each change is briefly described as commit titles, but some highlights are:
 - There is a new accessibility feature with the addition of a skip block at the beginning of every page that allows keyboard users to skip the nav and head directly to the main content.
 - Some buttons/icons have been given more context for screen readers with `aria-label`
 - Some of the descriptions of search results contained a long link that would cause the page to horizontally scroll on mobile sizes. This has been adjusted by allowing just results with overflow to scroll. (An example I found was titled "2023-0223 PUBLIC HEARING")

Connects #969 

## Testing Instructions
- On any page, tab over to the "skip to main content" link and hit enter (first element on the page, before the nav)
  - Confirm that focus changes to the main content of the page, and the page does not scroll the start of the content underneath the nav
  - Confirm that the link reveals itself only when it receives focus
- Check that the small styling changes listed in the commits improve upon or are similar to the original. These include:
  - nav item dropshadow and hover color
  - text in board member map on mobile sizes changed from 'tap a ward' to 'tap a region'
  - the order by buttons on the search page should now be lined up and spaced appropriately on various screen sizes
